autonomic-administration-plugin/src/main/java/br/com/autonomiccs/autonomic/administration/plugin/AdministrationAgent.java

```diff
@@ -105,8 +102,6 @@
      * @note spring integration configuration is in
      *       '/autonomic-administration-plugin/resources/META-INF/cloudstack/
      *       administration/spring-administration-context.xml' file.
-     *
-     * @param clusterId
```

Comment:
@pedro-martins, @GabrielBrascher note that we let quite a lot of this auto-generated documenting "@s"; we should either remove them or add some value to it (some explanation and or extra information).